src/main/java/uk/gov/hmcts/reform/unspec/config/properties/notification/NotificationsProperties.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.unspec.config.properties.notification;
+
+import lombok.Data;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+@Validated
+@Data
+public class NotificationsProperties {
+
+    @NotEmpty
+    private String govNotifyApiKey;
+
+    @NotNull
+    private EmailTemplates emailTemplates;
```

Comment:
if email templates is an enum then not sure this class is needed

Reply:
Email templates are not enum at the moment.

Reply:
amended